alerting/src/main/kotlin/org/opensearch/alerting/DocumentReturningMonitorRunner.kt

```diff
@@ -410,7 +491,47 @@ object DocumentReturningMonitorRunner : MonitorRunner {
         return response.hits
     }
 
+    private fun getMatchedQueries(
+        monitorCtx: MonitorRunnerExecutionContext,
+        docs: List<BytesReference>,
+        monitor: Monitor
+    ): SearchHits {
+        val percolateQueryBuilder = PercolateQueryBuilderExt("query", docs, XContentType.JSON)
+
+        val searchRequest = SearchRequest(ScheduledJob.DOC_LEVEL_QUERIES_INDEX)
```

Comment:
removed the safe check.

Reply:
Yes. the current logic does not take care of high availability. This can be improved in future.
Currently, if index doesnt exist, there will be no exceptions. only no search results will be returned.